### ISSUE TITLE
Add "copy to clipboard" button option for markup example.

### ISSUE THREAD

**Issue by patrickcate** [NoDivide/astrum#101]
Many styleguides and pattern libraries ([Bootstrap](http://getbootstrap.com) probably being the most obvious) have a convenient "copy to clipboard" button along with their markup examples, so users can easily copy and paste the examples. Adding this to Astrum would be a huge benefit for users, particularly those not as experienced working with HTML.

The [ZeroClipboard](http://zeroclipboard.org/) is probably the most popular library right now for adding this functionality, but relies on a hidden Flash component. There is also the pure javascript library[ clipboard.js](https://clipboardjs.com/) as well though.

![clipboard-example](https://cloud.githubusercontent.com/assets/6277206/23492464/87bdbfaa-fed2-11e6-8601-b5146cec8b67.png)

**Comment by RyanHavoc** [comment]
@patrickcate Absolutely. This is a feature we've been planning to add in. I've marked it to be worked on for the version 2.0.0 release. Thanks!
